server/verify.go

```diff
@@ -345,8 +346,24 @@ func VerifyGceTechnology(attestation *pb.Attestation, tech pb.GCEConfidentialTec
 			}
 			return VerifySevSnpAttestation(tee.SevSnpAttestation, snpOpts)
 		default:
-			return fmt.Errorf("TEE attestation is %v, expected an SevSnpAttestation", attestation.GetTeeAttestation())
+			return fmt.Errorf("TEE attestation is %v, expected a SevSnpAttestation", attestation.GetTeeAttestation())
 		}
+	case pb.GCEConfidentialTechnology_INTEL_TDX:
+		var tdxOpts *VerifyTdxOpts
+		var ok bool
+		tee, ok := attestation.TeeAttestation.(*pb.Attestation_TdxAttestation)
+		if !ok {
+			return fmt.Errorf("TEE attestation is %v, expected a TdxAttestation", attestation.GetTeeAttestation())
```

Comment:
I assume you want to log the type here? Try to use %T in the Errorf string

Reply:
Done